tests/bluetooth/bsim/mesh/tests_scripts/dfu/dfu_slot.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+# Copyright 2022 Nordic Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+source $(dirname "${BASH_SOURCE[0]}")/../../_mesh_test.sh
+
+# Test DFU with all variants of firmware effect
```

Comment:
From what I can see these tests are dependent of running in sequence to work as intended. Could you please add a comment that clarifies this. (Something like the comments added in `zephyr/tests/bluetooth/bsim/mesh/tests_scripts/persistence/access.sh`)

Reply:
Yes, I forgot to update this script with any appropriate comments